pkg/traceability/traceability_test.go

```diff
@@ -46,6 +46,8 @@ func createCentralCfg(url, env string) *config.CentralConfiguration {
 	authCfg.ClientID = "DOSA_1111"
 	authCfg.PrivateKey = "../transaction/testdata/private_key.pem"
 	authCfg.PublicKey = "../transaction/testdata/public_key"
+	cfg.PublishTrafficEvents = true
+	cfg.EventAggregationInterval = 30 * time.Second
```

Comment:
Same here increase default to 5 or 10 mins

Reply:
this is in a test